src/components/SendTransaction/ConfirmTransaction.tsx

```diff
@@ -224,6 +226,19 @@ export const ConfirmTransaction = ({
         </tbody>
       </TableEl>
 
+      {!formData.isAccountFunded && (
+        <InfoBlock>
+          The destination account doesn’t exist. A create account operation will
+          be used to create this account.{" "}
```

Comment:
❔ What does `{" "}` do? Is it to ensure there is whitespace between the text and the text link? I'm not concerned, I just want to learn.

Reply:
Exactly, @leighmcculloch. If it's just a space added without `{}`, it will be ignored in this case.